src/main/java/com/cisco/trex/stateful/model/stats/LatencyStats.java

```diff
@@ -45,19 +45,19 @@
      * see http://trex-tgn.cisco.com/trex/doc/cp_docs/api/json_fields.html#trex-latecny-field for detail description.
      */
     @JsonProperty("data")
-    public Map data;
+    public LatencyData data;
 
     @JsonProperty("name")
     public String name;
 
     @JsonProperty("type")
     public Integer type;
 
-    public Map getData() {
+    public LatencyData getData() {
```

Comment:
it's better to add @JsonProperty(key) annotation for all set/getXXX if it has corresponding field in json response.

Reply:
ok, I will adjust it.